Backend/src/routes/api.js

```diff
@@ -2,6 +2,7 @@ import { Router } from "express";
 import INITIAL_FRONTEND_DATA from "../../Data/cache_data.json";
 import INITIAL_SOLAR_CAR_DATA from "../../Data/dynamic_data.json";
 import DATA_FORMAT from "../../Data/sc1-data-format/format.json";
+import IMPORTED_CONSTANTS from "../../src/constants.json";
```

Comment:
No need to put imported in the name.
```suggestion
import CONSTANTS from "../../src/constants.json";
```